tools/build.js

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env node
+
+const commandRunner = require('./run-command');
+
+const build = 'npm run build';
+const build_db = 'npm run build -- --public-url=/"$GH-PAGES/"';
+
+commandRunner('mkdir -p deploy && rm -R deploy/', '.')
```

Comment:
Let's use the `mkdirp` node module.

Reply:
Also, let's use `rimraf` from npm to first `clean` the `dist/` dir, then you can do `mkdirp` to create it again.